runtime/mount/kernel.py

```diff
@@ -1084,6 +1085,38 @@ async def send_globals_summary(self) -> None:
 
         await self.send({"type": "globals_summary", "summary": summary})
 
+    async def upload_ldata(
+        self,
+        dst: str,
+        key: str,
+        viewer_id: str | None,
+        cell_id: str | None,
+        filename: str | None,
+    ) -> tuple[bool, str | None]:
+        df = self.k_globals[key]
```

Comment:
this might be a bit weird cuz the `__getitem__` goes through signals and tries to use the reactive context